README.md

```diff
@@ -4,4 +4,4 @@ You can include this mod in your modpack
 
 You can do video's with this fork of backpack mod
 
-My only request is to provide a link to this github if not its fine by me
+My only request is to provide a link to this GitHub repo. If not, it's fine by me
```

Comment:
this is not _strictly speaking_ a license, but still, dont change stuff like this!

Reply:
Is the proper process of fixing typos in such parts documented anywhere? I'm ready to document it and do the legwork on approving these changes.

Reply:
option 1 is to just leave it be. It is not our text and thus not for us to fix. If that is unsatisfying to you, option 2 would be to still leave it be, but frame it/put it in quotes and make clear these are runescapejons words and also link to his repo explicitely there. (though it being a fork there is a link on the page automatically)

Reply:
I'd agree with this train of thought if the original didn't have a license. However, it [does](https://github.com/runescapejon/Minecraft-Backpack-Mod/blob/MC1.7.x/LICENSE.md) have a license - LGPL 3. 

I mainly changed the line in README because I wanted to fix the "its" typo. If you want to keep it, sure, I can revert the changes.

Reply:
@chochem thank you for the clarification in PMs! I've addressed this point in 1faf2df.